.storybook/main.ts

```
@@ -0,0 +1,17 @@
+import type { StorybookConfig } from '@storybook/nextjs';
```

ESLint

[error] Replace `'@storybook/nextjs'` with `"@storybook/nextjs"`

(prettier/prettier)